tests/data/wgs1/README

```diff
@@ -0,0 +1,14 @@
+These are fastq files for a test sample.
+The _umi files were generated by removing the `/1` and `/2/ suffixes of the headers:
+zcat R1.fq.gz | sed 's/\/[1-2]//g' | gzip > R2.modified.fq.gz
+zcat R2.fq.gz | sed 's/\/[1-2]//g' | gzip > R1.modified.fq.gz
```

Comment:
Are the outputs of gzip switched here? Or did R1.fq.gz go into R2.modified.fq.gz?

Reply:
Ah no its a typo :P